config/config.go

```diff
@@ -50,9 +51,9 @@ func LoadConfigFile(path string) (*Config, error) {
 func LoadConfigDev() (*Config, chan struct{}, string, error) {
 	// start a vault dev instance
 	shutdownCh := initDevVaultCore()
-
+	time.Sleep(10 * time.Second)
 	// setup local vault instance with required mounts
-	err := SetupVault("http://127.0.0.1:8200", "goldfish")
+	err := setupVault("http://127.0.0.1:8200", "goldfish")
```

Comment:
`setupVault()` immediately tries to do post requests.

There is a race condition where if the goroutine hasn't completed setting up vault core, setupVault will fail because there is no active listener at port 8200. I'm assuming this doesn't happen on any of our dev computers because they are multi-core. Circle CI (afaik) is single-core, so it would be much more prone to these issues.